Dockerfile.jetty

```diff
@@ -26,8 +26,8 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/* && \
     /generate-jetty-start.sh
 
+ARG GRAPHVIZ_VERSION=14.0.1
```

Comment:
Not really sure how you bump your dependencies but this should be automated (at least the PR creation), e.g. by using [renovate](https://docs.renovatebot.com/)

Reply:
As a first step, we'd like to de-duplicate the GraphViz version in the Dockerfile: define it once, e.g.
`ARG GRAPHVIZ_VERSION=14.0.1` , and then reference it in each file so we don’t repeat the value three times.
If you think this approach looks good, feel free to open a new PR.

Longer term, the ideal would be to share this value between the `plantuml` and `plantuml-server` repos, but that's more involved, so let's go step by step.
We can revisit Renovate or a cross-repo strategy once the duplication is removed.

Thanks again for the help!